pkg/chains/storage/docdb/docdb_test.go

```diff
@@ -79,7 +80,8 @@ func TestBackend_StorePayload(t *testing.T) {
 
 			// Store the document.
 			opts := config.StorageOpts{Key: tt.args.key}
-			if err := b.StorePayload(ctx, tt.args.tr, sb, tt.args.signature, opts); (err != nil) != tt.wantErr {
+			trObj := objects.NewTaskRunObject(tt.args.tr, nil, ctx)
```

Comment:
We'd want these to play nice with pipeline runs as well in the future.

Reply:
Can you expand on this? Do you mean the other storage options?

Reply:
Yup! Just a note that we'll want to support the same storage options we do for TaskRuns for PipelineRuns as well.

Totally okay with starting with a 1-2 backends and adding more as we go though! 👍